packages/ckeditor5-dev-webpack-plugin/lib/replacetranslationcallsforonelanguage.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const {
+	getAllTranslations,
+	replaceFirstFunctionParameter,
+	createTranslator
+} = require( '@ckeditor/ckeditor5-dev-utils' );
+
+/**
+ * Replaces all function call parameters with translated strings for the t function.
+ *
+ * @param {Object} compiler Webpack compiler.
+ * @param {String} language Language code, e.g en_US.
+ */
+module.exports = function replaceTranslationCallsForOneLangauge( compiler, language ) {
+	const packageNames = new Set();
+
+	compiler.plugin( 'after-resolvers', ( compiler ) => {
+		compiler.resolvers.normal.plugin( 'before-resolve', ( obj, done ) => {
+			const match = obj.request.match( /@ckeditor[\\\/]([^\\\/]+)/ );
+
+			if ( match ) {
+				packageNames.add( match[ 1 ] );
+			}
+
+			done();
+		} );
+	} );
+
+	compiler.plugin( 'emit', ( options, done ) => {
+		const allTranslations = getAllTranslations( packageNames, language );
+
+		for ( const assetName in options.assets ) {
+			replaceTCalls( options.assets[ assetName ], allTranslations );
+		}
+
+		done();
+	} );
+};
+
+function replaceTCalls( assetContent, allTranslations ) {
```

Comment:
This, or something similar should be exposed by dev-utils. Not a meaningless functions like `replaceFirstFunctionParameter`. It must be fairly easy to build a plugin for bundler X based on our utils. The smaller the interface exposed by dev-utils, the better.